### DISCUSSION TITLE
Is CPU allocated to each container?

### DISCUSSION THREAD

**Discussion by tagoad** [AlexProgrammerDE/balena-minecraft-server#126] [Q&A]
So my server starts to struggle with 4+ people on it, but the cpu usage never shows above 40%. Is CPU power allocated to each container and the minecraft container is just maxed out even though there is more more avaliable?

**Comment by viedok** [comment]
Minecraft Java edition is strange sometimes. Maybe the CPU isn't the problem and your RAM is entirely used? Or maybe you aren't using a powerful enough power supply? Tons of things could cause this issue.
If you still want to resolve your issue you can contact me on Discord and I'll try my best to help you! taedium#6515
